.github/workflows/ci.yaml

```
@@ -176,7 +284,7 @@ jobs:
 
   publish-docs:
     name: Publish documentation website to GitHub pages
-    needs: ["nox", "test-installation"]
+    needs: ["nox", "nox-cross-arch", "test-installation"]
```

Also this, just to make the jobs graph look prettier :P

```suggestion
    needs: ["nox", "nox-cross-arch-all", "test-installation"]
```